addons/test_mail/tests/test_message_post.py

```diff
@@ -823,6 +823,53 @@ def test_message_post_w_attachments(self):
                         ]
         )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.addons.mail.models.mail_message_schedule')
+    def test_message_post_filter_main_attachments(self):
```

Comment:
Would name it 'test_message_post_w_attachments_filtering' to be in the same namespace of tests. Indeed as they are executed alphabetically, better keep the same ordering in the class. It helps finding them when they fail.